src/lib/evil/unposix/unistd.h

```diff
@@ -13,4 +25,10 @@
 UNIMPLEMENTED extern char *optarg;
 UNIMPLEMENTED extern int optind, opterr, optopt;
 
+UNIMPLEMENTED inline int getopt(int a, char * const b[], const char * c)
```

Comment:
isn't this a getopt.h function? (http://man7.org/linux/man-pages/man3/getopt.3.html